### PR TITLE
Release Synapse 1.19.1

### PR DESCRIPTION
**Pull request by babolivier**

### CODE REVIEW

File: gatsby/content/blog/2020/08/2020-08-27-synapse-1.19.1-released.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+date: '2020-08-27'
+title: Synapse 1.19.1 released
+categories:
+  - Releases
+author: Brendan Abolivier
+---
+
+Synappse 1.19.1 is a bug fix release to fix two bugs introduced in 1.19.0.
+
+Those two bugs were related to the new rate-limiter on room joins, which was introduced in 1.19.0.
```
Comment (anoadragon453) [inline gatsby/content/blog/2020/08/2020-08-27-synapse-1.19.1-released.mdx:11]:
```suggestion
Those two bugs are related to the new rate-limiter on room joins, which was introduced in 1.19.0.
```

File: gatsby/content/blog/2020/08/2020-08-27-synapse-1.19.1-released.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+The first bug caused Synapse to ignore exceptions to rate-limiting for application services.
+
+The second one prevented profile updates from being correctly propagated across rooms.
```
Comment (anoadragon453) [inline gatsby/content/blog/2020/08/2020-08-27-synapse-1.19.1-released.mdx:15]:
```suggestion
The second prevented profile updates from being correctly propagated across rooms.
```

File: gatsby/content/blog/2020/08/2020-08-27-synapse-1.19.1-released.mdx (hunk 1 above)
Comment (babolivier) [reply gatsby/content/blog/2020/08/2020-08-27-synapse-1.19.1-released.mdx:11]: I'd like to keep consistency across the post, if I'm talking about the bugs in the past later then it makes more sense that this sentence is also in the past

File: gatsby/content/blog/2020/08/2020-08-27-synapse-1.19.1-released.mdx (hunk 2 above)
Comment (babolivier) [reply gatsby/content/blog/2020/08/2020-08-27-synapse-1.19.1-released.mdx:15]: I find "the second one" easier to read imho

**Pull request merged**